### DISCUSSION TITLE
tutorials/bg-images-for-yolo/

### DISCUSSION THREAD

**Discussion by giscus** [Y-T-G/Yasins-Keep#10] [Giscus]
# tutorials/bg-images-for-yolo/

A simple guide to downloading background images to reduce false positives in object detection models.

https://y-t-g.github.io/tutorials/bg-images-for-yolo/

**Comment by vdt104** [comment]
What if my dataset doesn’t contain a lot of different background images? For example, I have a dataset of about 10,000 images, but the images are only from 10 different camera angles. In this case, if I randomly select an image and crop an area with no objects (to create around 1,000 false positive images), would that improve the results?

**Reply by Y-T-G** [reply]
That area of the image is already used as background during training. Any part of the image that's not labeled is considered background and is used for training. It's not the just part with the box that's used during training.

**Reply by vdt104** [reply]
Thank you for your answer. So, in my case, are there any feasible ideas to get more negative images?

**Reply by Y-T-G** [reply]
You can try running your model on your cameras and saving any detections that appear even though there are no objects.
